client/app/routes/application.js

```diff
@@ -25,5 +26,10 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
     } else if (this.get('currentDistrict.isInvalid')) {
       this.replaceWith('index');
     }
+  },
+  actions: {
+    trackEvent(eventName) {
+      this.get('intercom').trackEvent(eventName);
+    }
```

Comment:
This action adds an extra layer of indirection to tracking events – what do you think of just including the `intercom` service within each module (e.g. controller) that wants to track things?